widgetry/src/widgets/panel.rs

```diff
@@ -454,21 +454,7 @@ impl Panel {
     pub fn align(&mut self, horiz: HorizontalAlignment, vert: VerticalAlignment) {
         self.horiz = horiz;
         self.vert = vert;
-    }
-    pub fn align_above(&mut self, ctx: &mut EventCtx, other: &Panel) {
-        // Small padding
-        self.vert = VerticalAlignment::Above(other.top_level.rect.y1 - 5.0);
-        self.recompute_layout(ctx, false);
-
-        // Since we just moved things around, let all widgets respond to the mouse being somewhere
-        ctx.no_op_event(true, |ctx| assert_eq!(self.event(ctx), Outcome::Nothing));
-    }
-    pub fn align_below(&mut self, ctx: &mut EventCtx, other: &Panel, pad: f64) {
-        self.vert = VerticalAlignment::Below(other.top_level.rect.y2 + pad);
-        self.recompute_layout(ctx, false);
-
-        // Since we just moved things around, let all widgets respond to the mouse being somewhere
-        ctx.no_op_event(true, |ctx| assert_eq!(self.event(ctx), Outcome::Nothing));
+        // TODO Recompute layout and fire no_op_event?
```

Comment:
Correct. I realized `align_above` has no more callers, and layers<->minimap was the only caller of `align_below`. I'm tempted to also remove this `align`, since it's only used in the widgetry demo. Changing panel placement has some subtleties like this, and if we don't have to think through them yet, I like deleting code.